src/utility/ExportAgent.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+const rimraf = require('rimraf');
+const Dialogflow = require('dialogflow');
+const AdmZip = require('adm-zip');
+
+const projectId = process.env.DIALOGFLOW_PROJECT_ID;
+const file = process.argv[2] || './Agent';
+
+async function exportAgent() {
+  rimraf.sync(file);
```

Comment:
What does rimraf do?

Reply:
rimraf is a package that simulates the functionality of "rm -rf". Deletes the directory and any files in it.